openlibrary/templates/account/create.html

```diff
@@ -75,5 +78,31 @@ <h1>$_("Sign Up")</h1>
             <a href="javascript:history.go(-1);" class="smaller attn">$_("Cancel")</a>
         </div>
     </form>
-
-</div>
+    <style>
+        #password-visibility-toggle {
+            font-size: 14px;
+            margin-left: 5px;
+            cursor: pointer;
+        }
+
+    </style>
+    <script>
+        window.addEventListener("load", function() {
+            var passwordInput = document.getElementById("password");
+            var passwordVisibilityToggle = document.createElement("span");
+            passwordVisibilityToggle.id = "password-visibility-toggle";
+            passwordVisibilityToggle.innerHTML = '<i class="far fa-eye"></i>';
+            passwordVisibilityToggle.addEventListener("click", function(e) {
+                e.preventDefault();
+                if (passwordInput.type === "password") {
+                    passwordInput.type = "text";
+                    passwordVisibilityToggle.innerHTML = '<i class="far fa-eye-slash"></i>';
+                } else {
+                    passwordInput.type = "password";
+                    passwordVisibilityToggle.innerHTML = '<i class="far fa-eye"></i>';
+                }
+            });
+            passwordInput.parentNode.appendChild(passwordVisibilityToggle);
+        });
+    </script>
+</div>
```

Comment:
```suggestion
</div>

```
Add a trailing newline character to the end of this file.